torch/csrc/distributed/c10d/FileStore.cpp

```diff
@@ -313,7 +313,7 @@ FileStore::~FileStore() {
   // Clean up the file if number of references is 0.
   if (refCount == 0 && numWorkers_ >= 0 && numFinishedWorker >= numWorkers_) {
     // Best effort removal without checking the return
```

Comment:
Why this one? std::prefix is better, no?

Reply:
std::remove is for STL containers to remove elements . ::remove is a C file API to remove a file